wren-ui/src/components/pages/setup/dataSources/SQLServerProperties.tsx

```diff
@@ -88,6 +88,13 @@ export default function SQLServerProperties(props: Props) {
       >
         <Input placeholder="SQL Server database name" disabled={isEditMode} />
       </Form.Item>
+      <Form.Item
+        label="Enable Trust Server Certificate"
+        name="trustServerCertificate"
+        valuePropName="checked"
+      >
+        <Switch />
+      </Form.Item>
```

Comment:
Please help make adjustment to the property
- Default to checked
- Add `extra` prop to `Form.Item` with value: `This parameter is used to skip server certificate validation. If you are using a trusted certificate, you can disable it.`